features/search/default-queries-uitid.feature

```diff
@@ -0,0 +1,66 @@
+@sapi3
+Feature: Test the Search API v3 default queries from UiTID
+
+  Background:
+    Given I am using the UDB3 base URL
+    And I am using an UiTID v1 API key of consumer "uitdatabank"
+    And I am authorized as JWT provider v1 user "centraal_beheerder"
+    And I send and accept "application/json"
+
+  Scenario: Search for a place blocked by the default query
+    Given I create a place from "places/hemmekes.json" and save the "id" as "placeId"
+    And I am using the Search API v3 base URL
+    And I am not authorized
+    And I am using an UiTID v1 API key of consumer "sapi3KeyWithUitIdFilterForDiest"
+    And I am not using a x-client-id header
+    And I wait 2 seconds
```

Comment:
You could avoid this `wait`, if you use `I wait for the place with URL "%{placeUrl}" to be indexed` after the `Given`